src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1181 @@
+                0
+            ].conc_mass_phase_comp["Liq", "TDS"] / (1 - b.RR)
+
+        # Brine temperature
+        @self.Constraint(doc="Brine temperature")
+        def T_b_cal(b):
+            return (
+                b.brine_props[0].temperature
+                == b.distillate_props[0].temperature
+                + b.brine_props[0].boiling_point_elevation_phase["Liq"]
+            )
+
+        # Set alias for brine enthalpy and convert to kJ/kg
+        h_b = pyunits.convert(
+            self.brine_props[0].enth_mass_phase["Liq"], to_units=pyunits.kJ / pyunits.kg
+        )
+        # Set alias for brine volume (m3/hr) and mass (kg/s) flow rate
+        m_b = sum(
+            self.brine_props[0].flow_mass_phase_comp["Liq", j]
+            for j in self.brine_props.component_list
+        )
+        q_b = pyunits.convert(
+            self.brine_props[0].flow_vol_phase["Liq"],
+            to_units=pyunits.m**3 / pyunits.hour,
+        )
+
+        """
+        Add block for reject cooling water
+        """
+        self.cooling_out_props = self.config.property_package.state_block_class(
+            self.flowsheet().config.time,
+            doc="Material properties of cooling reject",
+            **tmp_dict
+        )
+
+        # Use the source water for cooling (Same salinity)
+        @self.Constraint(doc="Cooling reject salinity")
+        def s_cooling_cal(b):
+            return (
+                b.cooling_out_props[0].conc_mass_phase_comp["Liq", "TDS"]
+                == b.feed_props[0].conc_mass_phase_comp["Liq", "TDS"]
+            )
+
+        # Set alias for enthalpy and convert to kJ/kg
+        h_cool = pyunits.convert(
+            self.cooling_out_props[0].enth_mass_phase["Liq"],
+            to_units=pyunits.kJ / pyunits.kg,
+        )
+        # Assumption: the temperature of cooling reject is 3 degC lower than in the condenser
+        @self.Constraint(doc="Cooling reject temperature")
+        def T_cool_cal(b):
+            return (
+                b.cooling_out_props[0].temperature
+                == b.distillate_props[0].temperature - 3 * pyunits.K
+            )
+
+        # Set alias for cooling reject volume flow rate (m3/hr)
+        self.q_cooling = pyunits.convert(
+            self.cooling_out_props[0].flow_vol_phase["Liq"],
+            to_units=pyunits.m**3 / pyunits.hour,
+        )
+
+        """
+        Add block for heating steam
+        """
+        tmp_dict["parameters"] = self.config.property_package2
+        tmp_dict["defined_state"] = True
+
+        self.steam_props = self.config.property_package2.state_block_class(
+            self.flowsheet().config.time,
+            doc="Material properties of heating steam",
+            **tmp_dict
+        )
+
+        # Heating steam temperature is fixed at 70 C in this configuration
+        @self.Constraint(doc="Heating steam temperature")
+        def steam_temp(b):
+            return b.steam_props[0].temperature == 70 + 273.15
```

Comment:
Although 70C may be specific to the surrogate, I'd still opt for this value to be a parameter